tests/Agent/UnitTests/CompositeTests/AdaptiveSamplerTests.cs

```diff
@@ -38,7 +38,7 @@ public void BeforeEachTest()
         public void AfterEachTest()
         {
             _compositeTestAgent.Dispose();
-            _adaptiveSampler = null;
+            _adaptiveSampler.Dispose();
```

Comment:
Should we still set this value to null, in addition to the dispose call?

Reply:
I'm not sure, but it probably wouldn't hurt anything.